packages/website/src/App.tsx

```diff
@@ -76,8 +78,13 @@ function App(): React.ReactElement {
 
     return () => { axios.interceptors.request.eject(authInterceptor); };
   }, []);
+
+  const acceptSearchText = (textFromSearch: string) => {
+    setSearchText(textFromSearch);
+  };
+
   const userProviderValue = useMemo(() => ({ user, setUser }), [user, setUser]);
-  const NavBarMemo = useMemo(() => <NavBar />, [user, setUser]);
+  const NavBarMemo = useMemo(() => <NavBar acceptSearchText={acceptSearchText} />, [user, setUser, searchText, setSearchText]);
```

Comment:
Hey Kevin getting this error which I am trying to figure out why: 

```
Type '{ acceptSearchText: (textFromSearch: string) => void; }' is not assignable to type 'IntrinsicAttributes & { children?: ReactNode; }'.
  Property 'acceptSearchText' does not exist on type 'IntrinsicAttributes & { children?: ReactNode; }'.
```
Do you know what's going on here? Is it because of the wrapping of `NavBar`?